go.mod

```diff
@@ -2,6 +2,8 @@ module maistra.io/api
 
 go 1.15
 
+replace github.com/maistra/xns-informer => github.com/luksa/xns-informer v0.0.0-20221007103507-5488f0815e0f
```

Comment:
Need to remove this and update the actual dependency when https://github.com/maistra/xns-informer/pull/26 gets merged.